common/components/ui/UnitDisplay.tsx

```diff
@@ -43,7 +44,7 @@ const UnitDisplay: React.SFC<EthProps | TokenProps> = params => {
   const { value, symbol, displayShortBalance } = params;
 
   if (!value) {
-    return <span>???</span>;
+    return <Spinner size="x1" />;
```

Comment:
I'm not sure if we should have spinners when a user is purposefully using MEW from an offline machine.

Reply:
I would prefer explicit text saying that "Balances are not available when offline".

The same applies for equivalent values (and perhaps across the entire app?)